wire-ios-transport/Source/PushChannel/NativePushChannel.swift

```diff
@@ -58,25 +58,34 @@ class NativePushChannel: NSObject, PushChannelType {
     var consumerQueue: ZMSGroupQueue?
     var workQueue: OperationQueue
     var pingTimer: ZMTimer?
-
-    required init(scheduler: ZMTransportRequestScheduler,
-                  userAgentString: String,
-                  environment: BackendEnvironmentProvider,
-                  proxyUsername: String?,
-                  proxyPassword: String?,
-                  queue: OperationQueue) {
+    private let minTLSVersion: TLSVersion
+
+    required init(
+        scheduler: ZMTransportRequestScheduler,
+        userAgentString: String,
+        environment: BackendEnvironmentProvider,
+        proxyUsername: String?,
+        proxyPassword: String?,
+        minTLSVersion: String?,
+        queue: OperationQueue
+    ) {
         self.environment = environment
         self.scheduler = scheduler
         self.workQueue = queue
         self.proxyUsername = proxyUsername
         self.proxyPassword = proxyPassword
+        self.minTLSVersion = minTLSVersion.flatMap(TLSVersion.init) ?? .v1_2
```

Comment:
What about defining a global constant for the fallback version v1.2 ?

Reply:
So we can configure the fallback just once?

Reply:
yes

Reply:
done